parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -98,6 +98,24 @@ write_files:
   content: !!binary |
     {{CloudInitData "aptPreferences"}}
 
+{{if not .IsCoreOS}}
```

Comment:
if CoreOS + IPv6 isn't supported, can we add validation in https://github.com/Azure/aks-engine/pull/1424/files#diff-f45af0b14c127459636440a5981a6934R405 to fail fast when IPv6 is enabled with CoreOS? Then no need to check for CoreOS here.